index.html

```diff
@@ -2646,9 +2647,9 @@ <h2>Acknowledgements</h2>
 
       <p>
 The specification authors would like to thank the contributors to the
-W3C Decentralized Identifiers (DIDs) v1.0,
-W3C Verifiable Credential Data Integrity 1.0, and
-W3C Securing Verifiable Credentials using JOSE and COSE specifications
+<a href="https://www.w3.org/TR/did-core/">W3C Decentralized Identifiers (DIDs) v1.0</a>,
+<a href="https://www.w3.org/TR/vc-data-integrity/">W3C Verifiable Credential Data Integrity 1.0</a>, and
+<a href="https://www.w3.org/TR/vc-jose-cose/">W3C Securing Verifiable Credentials using JOSE and COSE</a> specifications
```

Comment:
We can use ReSpec triple bracket syntax for this:

```suggestion
[[[DID-CORE]]], [[[VC-DATA-INTEGRITY]]], and [[[VC-JOSE-COSE]]] specifications.
```

Reply:
I remembered why I didn't do that in the first place ... and should have had these URLs as the hrefs — because the acknowledgement isn't to the contributors of the moving "latest version" targets as the `[[[sytax]]]` will link to over time, but to the contributors of the *current* versions/drafts, as now linked below. I'll bow to whatever the group decides on this; this just makes the most sense to me.
```suggestion
<a href="https://www.w3.org/TR/2022/REC-did-core-20220719/">W3C Decentralized Identifiers (DIDs) v1.0</a>,
<a href="https://www.w3.org/TR/2024/CRD-vc-data-integrity-20240609/">W3C Verifiable Credential Data Integrity 1.0</a>, and
<a href="https://www.w3.org/TR/2024/CRD-vc-jose-cose-20240521/">W3C Securing Verifiable Credentials using JOSE and COSE</a> specifications
```

Reply:
Ah, good point, nevermind!